asf_welsh_energy_consultation/analysis/produce_plots_and_stats.py

```diff
@@ -255,7 +257,7 @@
     proportions_bar_chart(
         wales_df.loc[wales_df.CURRENT_ENERGY_RATING != "unknown"],
         "CURRENT_ENERGY_RATING",
-        "EPC ratings of all Welsh properties",
+        "EPC ratings of all Welsh EPC-registered properties",
```

Comment:
Might make more sense like this?  But i'll leave it up to you and Andy.
```suggestion
        "EPC ratings of all EPC-registered Welsh properties",
```